src/gui_elements.py

```diff
@@ -189,17 +192,18 @@ def render(self, screen: pygame.surface) -> None:
         """
         self.inner_text.position = center_relative_to(self.inner_text.position, self.hitbox)
         if isinstance(self.button_texture, pygame.color.Color):
-            pygame.draw.rect(screen, self.button_texture, self.hitbox, 0,
+            pygame.draw.rect(screen, self.button_texture, self.hitbox, width=0,
                              border_radius=self.border_radius)
         elif isinstance(self.button_texture, os.PathLike):
             # Кэшировние изображений, которые уже были зарендерины.
             if self.button_texture not in self._image_cache:
                 img = pygame.image.load(self.button_texture).convert_alpha()
                 if img.get_size() != (self.hitbox[2], self.hitbox[3]):
-                    img = pygame.transform.smoothscale(img, (self.hitbox[2], self.hitbox[3]))
+                    img = pygame.transform.smoothscale(img.convert_alpha(), (self.hitbox[2], self.hitbox[3]))
```

Comment:
```suggestion
                    img = pygame.transform.smoothscale(img, (self.hitbox[2], self.hitbox[3]))
```
Не увидел, что ты сверху и так это делаешь. Мог бы и сказать мне.
_(признайся, скопировал с какого-нибудь stackoverflow и даже не смотрел?)_